.github/workflows/ci.yaml

```diff
@@ -22,20 +22,18 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
+          - '3.12'
           - '3.11'
           - '3.10'
           - '3.9'
           - '3.8'
         airflow-version:
-          - '2.7.2'
-          - '2.6.3'
-          - '2.5.3'
-          - '2.4.3'
+          - '2.9.2'
+          - '2.8.4'
+          - '2.7.3'
```

Comment:
We'll probably want a policy for Airflow version support.

For Python versions it's easy: We aim to support whatever dbt supports, which [currently](https://github.com/dbt-labs/dbt-core/blob/main/core/setup.py#L5) has only a >=3.8. restriction.

For Airflow, I'd say we definitely want to test against [latest official minor version](https://docs.aws.amazon.com/mwaa/latest/userguide/airflow-versions.html#airflow-versions-official) (2.9.x), and ideally also [latest MWAA-supported version](https://docs.aws.amazon.com/mwaa/latest/userguide/airflow-versions.html#airflow-versions-official) (2.8.1). Astronomer pretty much supports new Airflow version as they come out, so we are covered there already.


Any other hosted service we should consider prioritizing? If not, I'd consider just supporting 2.9.2 and 2.8.1 here. There is a tradeoff between as much support as possible and CI times and dependency resolution. So, I'd try to keep the number of "tested" supported versions as small as possible.

Reply:
I would like to support Airflow 2.7.x as it is still in use by [Cloud Composer](https://cloud.google.com/composer/docs/concepts/overview)